v0/destinations/lambda/transform.js

```diff
@@ -60,6 +65,7 @@ const processRouterDest = inputs => {
     );
     return [respEvents];
   }
+  destination.Config.invocationType = "Event";
```

Comment:
Why we have hard coded this here?

Reply:
Setting invocation type as Asynchronous as we've decided.